src/lib.rs

```diff
@@ -7,10 +7,13 @@
 #![deny(missing_docs, clippy::integer_arithmetic)]
 
 use anyhow::{anyhow, ensure, Context, Result};
-use can_dbc::{Message, Signal, ValDescription, ValueDescription, DBC};
+use can_dbc::{Message, MultiplexIndicator, Signal, ValDescription, ValueDescription, DBC};
 use heck::{CamelCase, SnakeCase};
 use pad::PadAdapter;
-use std::io::{BufWriter, Write};
+use std::{
+    collections::{HashMap, HashSet},
```

Comment:
Makes sense I assume it weird when suddenly the generated code looks very different because of small changes in the dbc. I will change this accordingly.